mir-ci/mir_ci/test_drag_and_drop.py

```diff
@@ -39,28 +55,39 @@ class TestDragAndDrop:
             ("python3", APP_PATH, "--source", "text", "--target", "text", "--expect", "text"),
         ],
     )
-    async def test_source_and_dest_match(self, modern_server, app) -> None:
-        modern_server = DisplayServer(modern_server, add_extensions=VirtualPointer.required_extensions)
-        program = modern_server.program(apps.App(app))
-
-        robot_test_case = f"""Source and Destination Match
-            Sleep     {STARTUP_TIME}
-            Move Pointer To Absolute    40    40
-            Sleep     {A_SHORT_TIME}
-            Press LEFT Button
-            Sleep     {A_SHORT_TIME}
-            Move Pointer To Absolute    120    70
-            Sleep     {A_SHORT_TIME}
-            Move Pointer To Absolute    200    100
-            Sleep     {A_SHORT_TIME}
-            Release LEFT Button
+    async def test_source_and_dest_match(self, modern_server, app, tmp_path) -> None:
+        extensions = VirtualPointer.required_extensions + ScreencopyTracker.required_extensions
+        server_instance = DisplayServer(modern_server, add_extensions=extensions)
+        program = server_instance.program(apps.App(app))
+
+        robot_settings = dedent(
+            f"""\
+            Library    {MIR_CI_PATH}/robot_libraries/WaylandHid.py
+            Resource   {MIR_CI_PATH}/robot_resources/screencopy.resource
+        """
+        ).strip("\n")
+
+        robot_test_case = dedent(
+            f"""\
+            Source and Destination Match
+                Sleep     {STARTUP_TIME}
```

Comment:
Now that we wait for the source to be on screen, we shouldn't need that sleep?

Reply:
I'll check.